### PR TITLE
Finnish BIS (Independend Publisher) - Fixed response schema for Contact details

### PR DESCRIPTION
**Pull request by tpertila**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image

image

image

### CODE REVIEW

**Comment by tpertila** [comment]
Fixed response schema. Contact details section value was typed as integer but now fixed to be a string.

File: independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,716 @@
...
+                  "title": "Previous results URL"
+                },
+                "nextResultsUri": {
+                  "type": "string",
+                  "description": "nextResultsUri",
+                  "title": "Next results URL"
+                },
+                "exceptionNoticeUri": {
+                  "type": "string",
+                  "description": "",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json:77]: @tpertila Empty property values can be removed until unless you have a specific reason to keep them.

File: independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,716 @@
...
+                              "type": "string",
+                              "description": "name"
+                            },
+                            "type": {
+                              "type": "string",
+                              "description": "type"
+                            },
+                            "registrationDate": {
+                              "type": "string",
+                              "description": "registrationDate"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json:130]: @tpertila All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: independent-publisher-connectors/Finnish BIS/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Timo Pertilä",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Finnish BIS/apiProperties.json:6]: @tpertila Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Finnish BIS/apiProperties.json (hunk 3 above)
Comment (sriyen-msft) [reply independent-publisher-connectors/Finnish BIS/apiProperties.json:6]: @tpertila Can you please provide the publisher value in English language ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@tpertila Please resolve the branch conflicts of the PR to get it approved and merged.

**Comment by tpertila** [comment]
Scandic characters are removed now also from the apiProperties.json file.

**Comment by sriyen-msft** [comment]
@tpertila Can you please resolve the branch conflicts to get it approved and merged in the certification process?

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@tpertila Can you please resolve the branch conflicts to get it approved and merged in the certification process?

**Comment by tpertila** [comment]
Changed validation failed because of the correction itself. (ContactDetails value property type changed from integer (wrong) to string (proper)).

**Comment by tpertila** [comment]
"the type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/CompanyByBISCode/responses/200/schema/results/items/contactDetails/items/value/type Source: NewSwagger"

Currently, flow fails if there is any value on contactDetails value property. Becouse it has wrong datatype in original swagger file. Any change this can be merged?

**Comment by sriyen-msft** [comment]
@tpertila Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**